logmerge.py

```diff
@@ -72,6 +71,7 @@ def main_with_args(args, visitor=None, path_prefix=None, bar=None):
             visitor=visitor,
             wrap=args.wrap,
             wrap_indent=args.wrap_indent,
+            verbosity=args.verbose,
```

Comment:
better, imho, to keep the same name throughout the codebase, if possible... in this case, "verbose" and not rename it to verbosity?

Reply:
Agreed